docs/modules/operation/pages/quick-start/inventory.adoc

```diff
@@ -3,53 +3,50 @@
 = Step 3: Provision Inventory
 
 {page-component-title} installs with a base configuration that automatically begins service-level monitoring and reporting as soon as you identify and catalog (provision) the managed entities (nodes, IP interfaces, SNMP interfaces, and services) that you want to monitor.
+The default configuration also enables self-monitoring of the OpenNMS service, and discovers a single node with the interface `127.0.0.0` (see xref:deployment:core/getting-started.adoc#first-monitored-node[First monitored node]).
 
-{page-component-title} has three methods of provisioning:
-
-* xref:deep-dive/provisioning/auto-discovery.adoc[] (automatically add nodes).
-* xref:deep-dive/provisioning/directed-discovery.adoc[] (manually add nodes).
-* xref:development:rest/rest-api.adoc#rest-api[Asset import] through the RestAPI.
-
-Auto discovery is useful if you do not have a "source of truth" for your network inventory.
-Directed discovery is effective if you know your inventory, particularly if you have a smaller network (for example, 100–200 nodes).
+{page-component-title} has three methods of provisioning your network's inventory: xref:deep-dive/provisioning/auto-discovery.adoc[automatic discovery], xref:deep-dive/provisioning/directed-discovery.adoc[directed discovery], and xref:development:rest/rest-api.adoc[through the REST API].
 
 Regardless of the method, provisioning is an iterative process.
 You will need to fine-tune your results to exclude or add to what you monitor.
 
-This Quick Start guide presents a very basic example of auto discovery of nodes, through SNMP.
-Beyond this guide, you can complete additional tasks to provision your system:
+This Quick Start guide presents a very basic example using automatic discovery to provision nodes through SNMP.
 
-* Use plugins to integrate with external systems.
-* Specify more complex entity detection with OpenNMS detectors (ActiveMQ, DNS, FTP, JDBC, TCP, and so on).
-* Create policies to manage provisioning behavior.
+== SNMP in {page-component-title}
 
-Refer to xref:operation:deep-dive/provisioning/introduction.adoc[Provisioning] in the Deep Dive section for details about these and other provisioning-related tasks.
+SNMP is used in {page-component-title} to understand network and node topology, collect performance data, and enable SNMP service monitoring of nodes.
```

Comment:
```suggestion
{page-component-title} uses SNMP to understand network and node topology, collect performance data, and enable SNMP service monitoring of nodes.
```

Reply:
I'm not sure... @mhuot, would you be able to clarify?